src/genie/libs/parser/iosxe/show_nbar.py

```diff
@@ -0,0 +1,120 @@
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Or, Optional
+import re
+
+# ======================================================
+# Schema for 'show ip nbar protocol-discovery protocol'
+# ======================================================
+
+class ShowIpNbarDiscoverySchema(MetaParser):
+    schema = {
+        'interface': {
+            Any(): {
+                'protocol': {
+                    Any(): {
+                        'IN Packet Count': int,
+                        'OUT Packet Count': int,
+                        'IN Byte Count': int,
+                        'OUT Byte Count': int,
+                        'IN 5min Bit Rate (bps)': int,
```

Comment:
key name is lower case without space and brackets. so, please update to like `in_5min_bit_rate_bps`. others as well where applicable.

Reply:
Fixed, all changed to lower case. Thank you.

Reply:
confirmed the change